source/renderer/app/analytics/MatomoClient.ts

```diff
@@ -32,9 +32,10 @@ export class MatomoClient implements AnalyticsClient {
       _id: this.userId,
       action_name: pageTitle,
       url: this.getAnalyticsURL(),
-      [CPU_DIMENSION_KEY]: formatCpuInfo(this.environment.cpu),
+      [CPU_DIMENSION_KEY]: getShortCpuDescription(
+        this.environment.cpu[0]?.model || ''
+      ),
```

Comment:
I would have to return `'Unknown'` rather than `''`, as any unrecognized processor is marked as 'Unknown' in `getShortCpuDescription` (any non-recognized string including empty string).

What happens if someone decides to change the `Unknown` label for say `Unrecognized`? They would have to remember to change it in those two places.

If you insist, I can change the function signature to accept `undefined`

Reply:
Sorry, you're right about the `Unknown` CPU. The thing that I wanted to originally suggest is that you should check it first in `getShortCpuDescription` to avoid going through another `if` conditions that don't make sense to run for an empty string (minimal evaluation/optimization).

Now I'd suggest allowing `undefined` in func signature and check it in first line `if (!cpu) return 'Other';`.